modelscope/hub/api.py

```diff
@@ -433,6 +432,7 @@ def push_model(self,
                 remote_branch=revision)
             if tag is not None:
                 repo.tag_and_push(tag, tag)
+            logger.info(f'Successfully push folder {model_dir} to remote {model_id}.')
```

Comment:
Successfully push model folder {model_dir} to remote repo [{model_id}].

Reply:
done